test/extension/numeric/matrix3x2.cpp

```diff
@@ -16,8 +16,19 @@
 
 namespace gil = boost::gil;
 
-// FIXME: Remove when https://github.com/boostorg/core/issues/38 happens
-#define BOOST_GIL_TEST_IS_CLOSE(a, b, epsilon) BOOST_TEST_LT(std::fabs((a) - (b)), (epsilon))
+// Tolerance predicate for floating point comparison to use with BOOST_TEST_WITH
```

Comment:
Sweet, thanks for this!

Reply:
A basic sets of predicate should come with Boost.Core Lightweight Test IMHO...

Reply:
Yes, it would be useful